DirectXTex/DirectXTex_Desktop_2022.vcxproj

```diff
@@ -318,6 +318,7 @@
     <ClCompile Include="DirectXTexNormalMaps.cpp" />
     <ClCompile Include="DirectXTexPMAlpha.cpp" />
     <ClCompile Include="DirectXTexResize.cpp" />
+    <ClCompile Include="DirectXTexSwizzle.cpp" />
```

Comment:
We'll need to update all 9 vcxproj/filters sets to add this file. I can deal with that when you are ready.